data/monster/fey/twisted_pooka.lua

```diff
@@ -94,6 +94,8 @@ monster.loot = {
 	{name = "fairy wings", chance = 30100},
 	{name = "orange mushroom", chance = 5155},
 	{name = "shimmering beetles", chance = 591},
+	{name = "Prismatic Quartz", chance = 2500},
+	{name = "Crystal of Focus", chance = 60},
```

Comment:
Could you please just fix these 2 lines to keep the standard of all other monsters? If the loot is registered by the name, please use all letters with lower case. Thanks for your contribution.